src/lib/LUA/rx_devLUA.cpp

```diff
@@ -18,7 +18,14 @@ static const char *rxModes = "50Hz;60Hz;100Hz;160Hz;333Hz;400Hz;10kHzDuty;On/Off
 static struct luaItem_selection luaSerialProtocol = {
     {"Protocol", CRSF_TEXT_SELECTION},
     0, // value
-    "CRSF;Inverted CRSF;SBUS;Inverted SBUS;SUMD;DJI RS Pro;HoTT Telemetry",
+    "CRSF;Inverted CRSF;SBUS;Inverted SBUS;SUMD;DJI RS Pro;HoTT Telemetry;Spektrum Satellite",
+    STR_EMPTYSPACE
+};
+
+static struct luaItem_selection luaSatelliteSystem = {
+    {"Satellite System", CRSF_TEXT_SELECTION},
+    0, // value
+    "DSMX 11MS;DSMX 22MS;DSM2 11MS;DSM2 22MS",
```

Comment:
I dont know if Im a huge fan of an extra rx lua option just for this.  We could add each option to `luaSerialProtocol` the same way we have sbus and inverted sbus?

How common are all 4 protocols?

Reply:
I do not know much about SBUS but I believe "SBUS" and "Inverted SBUS" are two incompatible protocols. For example, if the FC expects "SBUS" but we choose "Inverted SBUS", it won't work. The main difference is, all these options are compatible (maybe with the exception of DSM2 22MS, I believe Betaflight separates these as Spektrum1024 and Spektrum 2048, the former for DSM2 22MS and the latter for the rest). So if you choose "DSMX 11MS" or "DSMX 22MS", it is still the same protocol and expected the work the same, just with a longer update duration.

The most commons are DSMX ones while the least commons ones are DSM2 ones. However, as far as I see, all DSMX Spektrum satellites are backwards compatible with DSM2.

Reply:
If we remove the luaSatelliteSystem lua option and add these to luaSerialProtocol, it means we can later use the 2 extra bits in the Config to expand serialProtocol from 16 to 64.  I think this be better long term.

Is there any harm in just having the 11ms option e.g. some hardware currently in use that does not accept 11ms?  22ms is painfully slow for ELRS.

Reply:
Having only DSMX 11MS would be fine probably. I just wanted to give users the freedom to emulate any valid configuration available out there, especially since it basically just takes a couple of if-statements once we have the base implementation. Also, I think 50Hz option is not too much faster than 22ms, doesn't change the fact that it is painfully slow though.